db/migrate/20171114150937_cleanup_installed_packages.rb

```diff
@@ -0,0 +1,55 @@
+class CleanupInstalledPackages < ActiveRecord::Migration
+  def up
+    create_table "katello_installed_packages_new", force: :cascade do |t|
+      t.string "name", limit: 255, null: false
+      t.string "nvra", limit: 255, null: false
+    end
+
+    create_table "katello_host_installed_packages_new", force: :cascade do |t|
+      t.integer "host_id",              null: false
+      t.integer "installed_package_id", null: false
+    end
+
```

Comment:
How about a unique index for the combination columns here. This should speed up the lookup.

Reply:
There arne't any lookups being done on this table?

Reply:
Good point. I guess add it on the ```katello_installed_packages_new```